Mage.Sets/src/mage/cards/g/Glamdring.java

```diff
@@ -85,7 +85,7 @@ public boolean apply(Game game, Ability source) {
     }
 
     @Override
-    public Effect copy() {
+    public GlamdringEffect copy() {
         return new GlamdringEffect();
```

Comment:
Guess I needed to be more clear: java boilerplate as follows:
![image](https://github.com/magefree/mage/assets/12538125/66600c5b-b638-43d5-ad3e-3650d9a129c7)

Reply:
Oh I see - I was not familiar with the pattern. Is it the case that whenever IntelliJ creates the `copy()` override, that I should use a copy constructor?

Reply:
Yep. And make sure to copy any inner fields if present.